profiler/include/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+install(DIRECTORY gz DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
+install(DIRECTORY ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
```

Comment:
this file can be deleted since these commands are in `profiler/CMakeLists.txt`

Reply:
54cbbac4